lib/analysis.js

```diff
@@ -273,6 +274,52 @@ function initNodeInfo(parent, kind, typeInfo) {
     };
 }
 
+function emitModuleRef(node, moduleData) {
+    var data = getDefinition(node.sourceFile, moduleData.end, moduleData.start);
+    var externalRepo = getExternalRepoInfo(data);
+
+    //Emission of refs for external modules, modules from node_modules
+    if (externalRepo) {
+        var ref = {
+            DefPath: formPathForModule(externalRepo),
+            Def: false,
+            File: node.sourceFile.name,
+            Start: moduleData.start,
+            End: moduleData.end
+        }
+        ref.DefRepo = externalRepo.repo;
+        ref.DefUnit = externalRepo.unit;
+        out.Refs.add(ref);
+    } else {
+        //Emission of simple module refs - for local files
+        if (data.origin && data.origin !== "node" && data.origin !== "commonjs" && data.origin !== "ecma5") {
```

Comment:
Will change, why not to move this function into utils? It will be easier to observe that such common functions were added?